source_modelling/sources.py

```diff
@@ -272,7 +277,11 @@ def dip(self) -> float:
         """float: The dip angle of the fault."""
         return np.degrees(np.arcsin(np.abs(self.bottom_m - self.top_m) / self.width_m))
 
-    # TODO: change this to take width and dip rather than projected width.
```

Comment:
Is this still important?

Reply:
I decided to make it a GitHub Issue instead because it's easier to track that way.

Reply:
But yeah it should do that.